src/main/kotlin/dev/arbjerg/lavalink/client/LavalinkNode.kt

```diff
@@ -104,7 +104,7 @@ class LavalinkNode(
             }
     }
 
-    fun createPlayer(guildId: Long) = PlayerUpdateBuilder(this, guildId)
+    fun updatePlayer(guildId: Long) = PlayerUpdateBuilder(this, guildId)
```

Comment:
How about?
```suggestion
    fun createOrUpdatePlayer(guildId: Long) = PlayerUpdateBuilder(this, guildId)
```

Reply:
I think `createOrUpdatePlayer` is stupidly long. Also this is doing a `PATCH` request

Reply:
It's a patch request that creates or updates the player, hence it should be called that

Reply:
since I won't be using this client, whatever lmao